pages/settings/index.html

```diff
@@ -85,6 +85,11 @@ <h3 data-string="settingsAdditionalFeaturesHeading"></h3>
 			<label for="checkbox-separate-titlebar" data-string="settingsSeparateTitlebarToggle"></label>
 		</div>
 
+		<div class="setting-section" hidden id='section-open-tabs-in-foreground'>
```

Comment:
I think the `hidden` attribute is the problem :)

Reply:
Yep, that made the preference show up. Not sure why though, I literally just copied `section-separate-titlebar` which contains `hidden`

Reply:
The separate titlebar pref is only supposed to be visible on Linux, so it starts out hidden and there's some JS that checks the operating system and then un-hides it.